ecommerce/views.py

```diff
@@ -258,6 +269,7 @@ def post(self, request, *args, **kwargs):
         )
 
 
+# pylint: disable=too-many-branches
```

Comment:
Is it worth moving this into a serializer at this point? you'd be able to move some of the validation into `def validate(self)` which would probably drop the branching level down below the lint threshold among other benefits

Reply:
Yes, probably better to do it now than later when it's more complicated